assets/js/src/core/modules/auth/components/forgot-password-form/forgot-password-form.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * This source file is available under the terms of the
+ * Pimcore Open Core License (POCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (https://www.pimcore.com)
+ *  @license    Pimcore Open Core License (POCL)
+ */
+
+import { Form } from '@Pimcore/components/form/form'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { Alert, Button, Flex, FormKit } from '@sdk/components'
+import { Input } from 'antd'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useAuthentication } from '../../hooks/use-authentication'
+
+interface ForgetPasswordForm {
+  username: string
+}
+
+interface ForgotPasswordFormProps {
+  onGetBack: () => void
+}
+
+export const ForgotPasswordForm = ({ onGetBack }: ForgotPasswordFormProps): React.JSX.Element => {
+  const { t } = useTranslation()
+  const [form] = Form.useForm<ForgetPasswordForm>()
+  const { resetPassword } = useAuthentication()
+  const [showSuccessMessage, setShowSuccessMessage] = useState<boolean>(false)
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  return (
+    <Flex
+      gap={ 16 }
```

Comment:
Would be better to use one of our pre-defined gab sizes here (so that the token gets applied)